R/s3functions.R

```diff
@@ -36,11 +36,20 @@ plot.diversityEstimates <- function(x, ...) {
   }
   xx <- args$xx
   h0 <- args$h0
-  
-  lci <- dv[[h0]] - 2*sqrt(dv[[paste(h0, "-variance", sep = "")]])
-  uci <- dv[[h0]] + 2*sqrt(dv[[paste(h0, "-variance", sep = "")]])
-  df <- data.frame("names" = names(dv[[h0]]), 
-                   "h0" = dv[[h0]], lci, uci, dv$X)
+  if (h0 %in% c("shannon", "simpson")) {
```

Comment:
Ultimately I think `plot.diversityEstimates` should be deprecated in favour of `breakaway::plot.alphaEstimates`, so don't worry too much about this function